ibm/service/vpc/resource_ibm_is_vpn_server.go

```diff
@@ -530,7 +530,7 @@ func resourceIBMIsVPNServerCreate(context context.Context, d *schema.ResourceDat
 	return resourceIBMIsVPNServerRead(context, d, meta)
 }
 
-func isWaitForVPNServerStable(context context.Context, sess *vpcv1.VpcV1, d *schema.ResourceData, timeout time.Duration) (interface{}, error) {
+func isWaitForVPNServerStable(context context.Context, sess *vpcv1.VpcV1, d *schema.ResourceData, vpnServer *vpcv1.VPNServer, timeout time.Duration) (interface{}, error) {
```

Comment:
Where are you using this vpnServer value which is passed to this function? Why you want to pass this value if its not used?

Thank you 
Sunitha

Reply:
added the missing part